### ISSUE TITLE
Allow multiple regressors to be specified at once

### ISSUE THREAD

**Issue by nicholst** [NISOx-BDI/SwE-toolbox#25]
When a design matrix is created and ready, it is very cumbersome to enter it column-by-column.

Should have an option to enter multiple columns at once, and then enter an array of names, one per column.

**Comment by nicholst** [comment]
Grab example code from https://github.com/spm/spm12/blob/master/config/spm_cfg_factorial_design.m#L744-L777 ... adding a "Multiple Covariates" option.

Additionally, add an item for "Names" so that, e.g., if a text file is supplied, the names can be supplied.

**Comment by TomMaullin** [comment]
Hi @nicholst ,

I was just wondering if you have any example `.mat`/`.txt` files I could use look at as an example of the code I should add?

**Comment by nicholst** [comment]
Any plain text file that can be parsed by Matlab into a matrix with nScan rows and any number of columns.  I.e. anything that works as 
```
X = load('Vars.dat');
```
where, e.g. you could generate `Vars.dat` like:
```
nScan=30;
nVar = 5;
Vars = rand(nScan,nVar);
Nms = strcat('Var_',num2str([1:nVar]'));
save Vars.dat Vars -ascii 
```

Now sure how you are handling the names... they could come from a plain text file or you could require them in a cell array like `Nms`.

**Comment by TomMaullin** [comment]
Ah I see! Thank you... so I should assume they are all in one matrix? I originally thought we were thinking about data generated in a way something along the lines of:

```
Vars = struct('Var1', [1 1 1], 'Var2', [0 1 2])

save('Vars.mat', 'Vars')
```
(supposing there were 3 subjects).

The only reason I thought this was that each variable could then have it's associated name (`Var1`, `Var2` etc) stored alongside it.

**Comment by TomMaullin** [comment]
This issue has now been addressed by PR #45
